system-x/services/prometheus-metrics/src/main/java/software/tnb/prometheus/metrics/service/PrometheusMetrics.java

```diff
@@ -0,0 +1,42 @@
+package software.tnb.prometheus.metrics.service;
+
+import software.tnb.common.config.OpenshiftConfiguration;
+import software.tnb.common.openshift.OpenshiftClient;
+import software.tnb.common.service.Service;
+import software.tnb.prometheus.metrics.validation.PrometheusMetricsValidation;
+
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.auto.service.AutoService;
+
+import io.fabric8.openshift.api.model.Route;
+
+@AutoService(PrometheusMetrics.class)
+public class PrometheusMetrics implements Service {
```

Comment:
this will only work on openshift, since the prometheus is deployed there by default.

please split it into `LocalPrometheusMetrics` and `OpenshiftPrometheusMetrics` - you don't need to implement the local part